autonbox/merge_multipredictions.py

```diff
@@ -0,0 +1,56 @@
+import os
+import typing
+import numpy as np
+import pandas as pd
+
+from d3m import container
+from d3m import utils as d3m_utils
+from d3m.metadata import base as metadata_base, hyperparams
+from d3m.primitive_interfaces.base import CallResult
+from d3m.primitive_interfaces.transformer import TransformerPrimitiveBase
+from d3m.primitives.data_preprocessing.horizontal_concat import DSBOX
+
+import autonbox
+
+Inputs = container.List
+Outputs = container.DataFrame
+
+class MergePartialPredictionsPrimitive(TransformerPrimitiveBase[Inputs, Outputs, hyperparams.Hyperparams]):
+    """
+        Merge predictions of multiple models
+        Useful if model do not produce predictions for each points and that it is necessary
+        to merge those predictions (the first non nan will be returned)
+    """
+
+    metadata = metadata_base.PrimitiveMetadata({
+        'id': '1cc95f70-0716-11ea-9762-3dd2bb86dde8',
+        'version': '0.1',
```

Comment:
Use three part naming conversion for version: i.e. [Major].[Minor].[Patch]
`'version':'0.1.0'`